src/modules/csm/csm.py

```diff
@@ -237,18 +244,27 @@ def calculate_distribution(self, blockstamp: ReferenceBlockStamp) -> tuple[int,
         return distributed, shares
 
     def stuck_operators(self, blockstamp: ReferenceBlockStamp) -> Iterable[NodeOperatorId]:
+        stuck: set[NodeOperatorId] = set()
         l_epoch, _ = self.current_frame_range(blockstamp)
         l_ref_slot = self.converter(blockstamp).get_epoch_first_slot(l_epoch)
         # NOTE: r_block is guaranteed to be <= ref_slot, and the check
         # in the inner frames assures the  l_block <= r_block.
-        return self.w3.csm.get_csm_stuck_node_operators(
+        l_blockstamp = build_blockstamp(
             get_next_non_missed_slot(
                 self.w3.cc,
                 l_ref_slot,
                 blockstamp.slot_number,
-            ).message.body.execution_payload.block_hash,
-            blockstamp.block_hash,
+            )
+        )
+        no_by_module = self.w3.lido_validators.get_lido_node_operators_by_modules(l_blockstamp)
+        stuck.update(no.id for no in no_by_module.get(self.module_id) if no.stuck_validators_count > 0)
```

Comment:
`get` can return None.
I propose here to do `no_by_module[self.module_id]`

Reply:
fixed. now it rises `InconsistentData` exception when no digests were found